Makefile

```diff
@@ -14,3 +14,7 @@ coverage:
 lint:
 	# sudo apt-get install python-flake8
 	$(PYTHON) -m flake8 .
+
+.PHONY: install-dev
+install-dev:
+	ln -s $(shell pwd)/txjuju /usr/local/lib/python2.7/dist-packages/txjuju
```

Comment:
hacky, but we could test for existence first and noop if it already exists?
test -e /usr/local/lib/python2.7/dist-packages/txjuju || ln -s ...